mangaki/mangaki/utils/anidb.py

```diff
@@ -208,14 +209,24 @@ def get_or_update_work(self,
                     "anidb_creator_id": creator_id
                 })
 
+        # Handling of tags
+        tags = {}
+        for tag_node in all_tags.find_all('tag'):
+            tag_title = str(tag_node.find('name').string).strip()
+            tag_id = tag_node.get('id')
+            tag_weight = tag_node.get('weight')
+            tag_verified = tag_node.get('verified')
```

Comment:
## An isolated remark:

If you know that the `tag` is of type `bool`, whenever you can, it is useful to coerce it into a real boolean rather than an `str`:

- The usual way to coerce some value into boolean is to use `bool`.
- **But:** it won't work with `str`, why?

There are multiple ways to convert "a string boolean" (depends on your data), here are some pointers:

- `tag_verified.lower() == "true"` (simple & quick test)
- `tag_verified.lower() in ('true', 'yes', 'y', '1', …)` (more complex tests for command line tools for example.)